hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/handle/DispatchProcessorTest.java

```diff
@@ -414,6 +437,7 @@ void consGasExhaustedWaivesServiceFee() throws ThrottleException {
         verify(recordBuilder).status(CONSENSUS_GAS_EXHAUSTED);
         verify(feeAccumulator).chargeFees(PAYER_ACCOUNT_ID, CREATOR_ACCOUNT_ID, FEES);
         verify(feeAccumulator).chargeFees(PAYER_ACCOUNT_ID, CREATOR_ACCOUNT_ID, FEES.withoutServiceComponent());
+        verify(opWorkflowMetrics).incrementThrottled(CONTRACT_CALL);
```

Comment:
This test and `consGasExhaustedForEthTxnDoesExtraWork()` are the only tests I see in here that are actually testing `incrementThrottled`, and they're both contract-related. It would be nice to see other more 'native' transaction types also work. The majority of the lines added to this file read 
```
verify(opWorkflowMetrics, never()).incrementThrottled(any());
```
i.e. verifying the throttle _isn't_ incremented, which is important, but may not be sufficient.

Likewise for the IngestCheckerTest, it would be nice to see more assertions about actually incrementing the throttle appropriately.

Reply:
I added a new test in DispatchProcessorTest for a native transaction type.
The other files already have 100% test coverage on the patch. I think there could be better test coverage of throttle logic esp in OpWorkflowMetricsTest. But, that is outside the scope of this PR.